modules/data.land/R/match_species_id.R

```diff
@@ -68,18 +68,49 @@ match_species_id <- function(input_codes, format_name = 'custom', bety = NULL, t
                                             suffix = c('.translation_table', ''))
         }
     } else {
-        column <- formats_dict[format_name]
+      column <- formats_dict[format_name]
+      if(!is.null(bety)){
+        # query BETY
         filter_cri <- lazyeval::interp(~ col %in% codes, 
                                        col = as.name(column),
                                        codes = input_codes)
         translation <- dplyr::tbl(bety, 'species') %>%
-            dplyr::filter_(filter_cri) %>%
-            dplyr::select_('bety_species_id' = 'id', 'genus', 'species',
-                           'input_code' = column) %>%
-            dplyr::collect()
+          dplyr::filter_(filter_cri) %>%
+          dplyr::select_('bety_species_id' = 'id', 'genus', 'species',
+                         'input_code' = column) %>%
+          dplyr::collect()
+        
+      }else{
+        # use traits package
+
+        # can call traits::betydb_query one at a time?
+        # reduce the number of calls
+        translation <- data.frame(input_code = unique(input_codes),
+                           bety_species_id  = rep(NA, length(unique(input_codes))),
+                           genus            = rep(NA, length(unique(input_codes))),
+                           species          = rep(NA, length(unique(input_codes))),
+                           stringsAsFactors = FALSE)
+                    
+        for(i in 1:nrow(unique.tmp)){
+          foo <- eval(parse(text =paste0("traits::betydb_query(", 
```

Comment:
@ashiklom I want to use the `traits` package in this function but the development version (`install_github("ropensci/traits")`) how to import it then?